source/main/gui/panels/GUI_TopMenubar.cpp

```diff
@@ -40,7 +40,7 @@ void RoR::GUI::TopMenubar::Update()
 
     const char* sim_title = "Simulation"; // TODO: Localize all!
     Str<50> actors_title;
-    actors_title << "Actors (" << App::GetSimController()->GetNumPlayableActors() << ")";
+    actors_title << "Actors (" << App::GetSimController()->GetPlayableActors().size() << ")";
```

Comment:
As much as I like the API more, this means 2 `std::vector<>`s are allocated every frame. Not a real performance issue, just saying.

Reply:
It's just a temporary fix. I think the BeamFactory should simply keep track of the amount of (playable) actors and provide a getter method for the actual number.